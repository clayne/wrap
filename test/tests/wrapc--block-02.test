wrapc | /dev/null | | wrapc-block-02.txt | 0
