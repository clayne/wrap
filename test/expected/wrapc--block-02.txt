        # We want to resolve the symbolic link of ONLY src_path and not ALL the
        # symbolic links in the entire path, hence we can't use realpath().
	##
        new_src_path = os.readlink( src_path )
